main.tf

```diff
@@ -375,6 +375,11 @@ data "aws_iam_policy_document" "ecs_tasks" {
       type        = "Service"
       identifiers = compact(distinct(concat(["ecs-tasks.amazonaws.com"], var.trusted_principals)))
     }
+
+    principals {
+      type        = "AWS"
+      identifiers = compact(distinct(var.trusted_entities))
```

Comment:
This code won't work if `trusted_entities` is empty. Please rewrite it to use `dynamic`.

Reply:
That is indeed a good reason :) Thanks for the quick response. I will rewrite it

Reply:
@antonbabenko I wasn't sure how to contribute to this PR so i created another. Hope that's OK.

https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/193